test/testdata/log_cni_plugin.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# this simple cni plugin just log stdin into file
+
+config=$(cat /dev/stdin)
```

Comment:
I think you might want to extend this logic in the [existing](https://github.com/cri-o/cri-o/blob/main/test/cni_plugin_helper.bash) file.